Kitodo/src/test/java/org/kitodo/production/services/catalogimport/ImportServiceTest.java

```diff
@@ -21,23 +21,23 @@
 public class ImportServiceTest {
```

Comment:
Recommendation: You have added the following line here wouldn't be necessary also to test the FTP for `false`? Or is this not necessary when OAI is `true`?
```
        if (SearchInterfaceType.FTP.name().equals(configuration.getInterfaceType())) {
            return false;
        }
```

Reply:
Added another test for FTP configuration